test/Buddy/functional/InsertQueryTest.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+/*
+ Copyright (c) 2022, Manticore Software LTD (https://manticoresearch.com)
+
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License version 2 or any later
+ version. You should have received a copy of the GPL license along with this
+ program; if you did not, you can find it at http://www.gnu.org/
+ */
+
+use Manticoresearch\BuddyTest\Trait\TestFunctionalTrait;
+use PHPUnit\Framework\TestCase;
+
+class InsertQueryTest extends TestCase {
+
+	use TestFunctionalTrait;
+
+	public function testSqlInsertQueryOk(): void {
+		echo "\nTesting the execution of SQL insert query to a non-existing table\n";
+		$query = 'INSERT into test(col1,col2) VALUES(1,2) ';
+		exec("mysql -P8306 -h127.0.0.1 -e '$query' 2>&1", $out, $res);
```

Comment:
Do you think it is a good idea to test like this? What if we have no MySQL client?
Maybe it's better to test by standard endpoints of Manticore? Anyway, it works the same way if I'm right.

Reply:
I feel we need to test user's work via MySQL console too if we target the test coverage as full as possible.  But yes, it's going to require adding MySQL to the testing Docker image; it's absent now as I understand.  I think we can also add the check for MySQL availability to tests to handle such situation.

Reply:
Okay, I guess it's good point to test with different tools to be sure that everything works fine